### PR TITLE
Cu 31gy3qt blockchain integrations picasso transfer page usdt statemine to picasso mamali

### PR DESCRIPTION
**Pull request by easteregg**
## Issue
https://app.clickup.com/t/31gy3qt

### CODE REVIEW

**Comment by ComposableFiB0t** [comment]
Task linked: [CU-31gy3qt Blockchain Integrations: Picasso transfer page: USDT statemine to Picasso](https://app.clickup.com/t/31gy3qt)

File: frontend/apps/picasso/defi/polkadot/pallets/Assets/statemine.ts [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+type StatemineAssetDetail = {
+  owner: string;
+  issuer: string;
+  freezer: string;
+  supply: number;
+  deposit: number;
+  minBalance: number;
+  isSufficient: true;
+  accounts: number;
+  sufficients: number;
```
Comment (Martin-Kiss) [inline frontend/apps/picasso/defi/polkadot/pallets/Assets/statemine.ts:24]:
```suggestion
  sufficient: number;
```
Nit spelling, can we rename this without breaking anything?

File: .config/dictionaries/technical.dic [hunk 2]
```diff
@@ -282,6 +283,7 @@ RUSTFLAGS
 rustfmt
 RUSTSEC # RUSTSEC-XXXX-XXXX is id of RustSec Advisory Database
 rustup
+sufficients
```
Comment (Martin-Kiss) [inline .config/dictionaries/technical.dic:286]:
```suggestion
```
Nit typo, failing spellcheck CI. Please confirm if this addition is necessary for syntax to work in [frontend/apps/picasso/defi/polkadot/pallets/Assets/statemine.ts](https://github.com/ComposableFi/composable/pull/2217/files#diff-350510d2f5d2f6d66add1a00507db201affc4343f8b1af4247a6f14b25876833)

File: frontend/apps/picasso/defi/polkadot/pallets/Assets/statemine.ts (hunk 1 above)
Comment (easteregg) [reply frontend/apps/picasso/defi/polkadot/pallets/Assets/statemine.ts:24]: this is coming from the api. we unfortunately need to keep this.

Comment (Martin-Kiss) [reply frontend/apps/picasso/defi/polkadot/pallets/Assets/statemine.ts:24]:
```suggestion
  sufficients: number; // Cspell:ignore sufficients
```
This solves it for the entire file. If we add more repeated use of `sufficients` outside of this file we should add it to the dictionary in the future.

File: frontend/apps/picasso/stores/defi/polkadot/transfers/transfers.ts [hunk 3]
```diff
@@ -250,6 +253,7 @@ export const createTransfersSlice: StoreSlice<TransfersSlice> = (set, get) => ({
           transferExtrinsic === null ||
           get().transfers.destinationMultiLocation === null
         ) {
+          console.warn("Shush");
```
Comment (andresmechali) [inline frontend/apps/picasso/stores/defi/polkadot/transfers/transfers.ts:256]: leftover?

File: frontend/packages/shared/defi/unit.ts [hunk 4]
```diff
@@ -8,20 +8,21 @@ export function toChainIdUnit(value: number | BigNumber, decimalPlaces = 12) {
   return bigNumberValue.multipliedBy(10 ** decimalPlaces);
 }
 
-export function fromChainIdUnit(value: number | BigNumber | BigInt, decimalPlaces = 12) {
-  return (BigNumber.isBigNumber(value) ? value : new BigNumber(value.toString())).dividedBy(
-    10 ** decimalPlaces
-  );
+export function fromChainIdUnit(
```
Comment (andresmechali) [inline frontend/packages/shared/defi/unit.ts:11]: Historical values coming from Subsquid are currently already doing this conversion, so you will not always need it.

**Review by andresmechali — APPROVED** [review]

**Review by saadjhk — APPROVED** [review]

**Review by andyjsbell — APPROVED** [review]
lgtm

File: frontend/apps/pablo/defi/polkadot/Assets.ts [hunk 5]
```diff
@@ -24,6 +24,7 @@ export const Assets: {
     supportedNetwork: {
```
[the remark this answers is not in the document]
Comment (easteregg) [reply frontend/apps/pablo/defi/polkadot/Assets.ts:24]: Not from the latest main, should I remove it?

File: frontend/packages/shared/defi/unit.ts (hunk 4 above)
Comment (easteregg) [reply frontend/packages/shared/defi/unit.ts:11]: Thanks for letting me know. But this is a utility function we use to convert all values from on chain.

File: frontend/apps/pablo/defi/polkadot/Assets.ts (hunk 5 above)
[the remark this answers is not in the document]
Comment (saadjhk) [reply frontend/apps/pablo/defi/polkadot/Assets.ts:24]: oh its pablo, its fine then @easteregg

**Pull request merged**